app/services/analytics_events.rb

```diff
@@ -465,6 +465,134 @@ def idv_doc_auth_exception_visited(step_name:, remaining_attempts:, **extra)
     )
   end
 
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [Integer] attempts
+  # @param [Integer] remaining_attempts
+  # @param [String] user_id
+  # @param [String] flow_path
+  # The document capture  image uploaded was locally validated during the IDV process
+  def idv_doc_auth_submitted_image_upload_form(
+    success:,
+    errors:,
+    remaining_attempts:, flow_path:, attempts: nil,
```

Comment:
let's break these on to separate lines for consistency
```suggestion
    remaining_attempts:,
    flow_path:,
    attempts: nil,
```